docs/sg_execution_times.rst

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
Shouldn't be committing this I think?

Reply:
Good catch -- this will be removed in #43 (updated the `.gitignore`)

Reply:
> Good catch -- this will be removed in #43 (updated the `.gitignore`)

Yes, `/.gitignore` has been updated. But there still is `docs/sg_execution_times.rst` there.

Reply:
> > Good catch -- this will be removed in #43 (updated the `.gitignore`)
> 
> Yes, `/.gitignore` has been updated. But there still is `/docs/sg_execution_times.rst` there.

#45 is posted to fix it.

Reply:
Fixed in  #47.